### PR TITLE
docs: update Portuguese translation

### PR DESCRIPTION
**Pull request by gilhrpenner**
Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

Created the missing pages for the deploy section as well as reordered  the content of a few pages that had changes.

### CODE REVIEW

**Comment by c-ehrlich** [comment]
@claudiofreitas
@minsk-dev
@Sn0wye
@victoriaquasar

File: www/src/pages/pt/deployment/docker.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ CMD ["node", "server.js"]
 >
 > - _A emulação de `--platform=linux/amd64` pode não ser necessária após a mudança para o Node 18._
 > - _Consulte [`node:alpine`](https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine) para entender por que `libc6-compat` pode ser necessário._
+> - _Usando imagens baseada em Alpine 3.17 [pode causar problemas no Prisma](https://github.com/t3-oss/create-t3-app/issues/975). Setting `engineType = "binary"` corrige esse problema no Alpine 3.17, [but has an associated performance cost](https://www.prisma.io/docs/concepts/components/prisma-engines/query-engine#the-query-engine-at-runtime)._
```
Comment (victoriaquasar) [inline www/src/pages/pt/deployment/docker.md:133]:
```suggestion
> - _Usando imagens baseadas em Alpine 3.17 [pode causar problemas no Prisma](https://github.com/t3-oss/create-t3-app/issues/975). Realizando a configuração: `engineType = "binary"` corrige esse problema no Alpine 3.17, [mas apresenta um custo de performance adicional](https://www.prisma.io/docs/concepts/components/prisma-engines/query-engine#the-query-engine-at-runtime)._
```

File: www/src/pages/pt/deployment/netlify.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+title: Netlify
+description: Deploy no Netlify
+layout: ../../../layouts/docs.astro
+---
+
+Netlify é um provedor de implantação alternativo em uma linha semelhante ao Vercel. Veja [`ajcwebdev/ct3a-netlify`](https://github.com/ajcwebdev/ct3a-netlify) para um repositório de exemplo baseado neste documento.
```
Comment (victoriaquasar) [inline www/src/pages/pt/deployment/netlify.md:7]:
```suggestion
Netlify é um provedor de computação em nuvem remota com suporte a frameworks modernos de desenvolvimento, semelhante ao Vercel. Veja [`ajcwebdev/ct3a-netlify`](https://github.com/ajcwebdev/ct3a-netlify) para um repositório de exemplo baseado neste documento.
```

File: www/src/pages/pt/deployment/netlify.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+## Por que hospedar no Netlify
+
+A sabedoria convencional diz que o Vercel tem suporte superior ao Next.js porque o Vercel desenvolve o Next.js. Eles têm um interesse em garantir que a plataforma seja otimizada para o melhor desempenho e DX com o Next.js. Para a maioria dos casos de uso, isso será verdade e não fará sentido desviar do caminho padrão.
```
Comment (victoriaquasar) [inline www/src/pages/pt/deployment/netlify.md:11]:
```suggestion
Comumente, a plataforma Vercel tem um suporte mais refinado ao Next.js porque o a mesma desenvolve o framework. Eles têm um interesse em garantir que a plataforma seja otimizada para o melhor desempenho e DX com o Next.js. Para a maioria dos casos de uso, isso será verdade e não fará sentido usar outras alternativas.
```

File: www/src/pages/pt/deployment/netlify.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+Também existe um sentimento comum de que muitos recursos do Next.js são suportados apenas no Vercel. Embora seja verdade que os novos recursos do Next.js serão testados e suportados no Vercel no momento do lançamento por padrão, também é o caso de que outros provedores, como o Netlify, [implementarão e lançarão rapidamente suporte](https://www.netlify.com/blog/deploy-nextjs-13/) para [recursos Next.js estáveis](https://docs.netlify.com/integrations/frameworks/next-js/overview/).
```
Comment (victoriaquasar) [inline www/src/pages/pt/deployment/netlify.md:13]:
```suggestion
Também sabe-se que muitos recursos do Next.js são suportados apenas no Vercel. Embora seja verdade que os novos recursos do Next.js serão testados e suportados no Vercel no momento do lançamento por padrão, também é o caso de que outros provedores, como o Netlify, [implementarão e lançarão rapidamente suporte](https://www.netlify.com/blog/deploy-nextjs-13/) para [recursos Next.js estáveis](https://docs.netlify.com/integrations/frameworks/next-js/overview/).
```

**Comment by victoriaquasar** [comment]
> Closes #
> 
> ## ✅ Checklist
> * [x]  I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
> * [x]  The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
> * [x]  I performed a functional test on my final commit
> 
> ## Changelog
> Created the missing pages for the deploy section as well as reordered the content of a few pages that had changes.

Which issue does it close?

File: www/src/pages/pt/usage/next-auth.md [hunk 5]
```diff
@@ -34,6 +34,29 @@ const User = () => {
...
+export const getServerSideProps: GetServerSideProps = async (ctx) => {
+  const session = await getServerAuthSession(ctx);
+  return {
+    props: { session },
+  };
+};
+
+const User = () => {
+  const { data: session } = useSession();
+  // NOTE: `session` wont have a loading state since it's already prefetched on the server
```
Comment (victoriaquasar) [inline www/src/pages/pt/usage/next-auth.md:54]: Traduzir comentário (não faz parte do código)

**Comment by gilhrpenner** [comment]
> 

I went over the existing issues and didn't see any for pt-BR translation. When I have some time tomorrow I can go over the guidelines and open an issue.

**Comment by Sn0wye** [comment]
A página da Netlify já foi criada, favor verificar.
Netlify page has already been created, please check.

**Review by c-ehrlich — DISMISSED** [review]
Thanks for the contribution and for taking the time to update so much of the docs.

We just merged a different translation for the Netlify page. Could you please resolve the merge conflict (feel free to modify the existing Netlify page as little or as much as you want, and then we'll have the other translators review it.

**Comment by gilhrpenner** [comment]
@c-ehrlich @Sn0wye fixed the merge conflict with the existing Netlify page.

File: www/src/pages/pt/deployment/index.astro [hunk 6]
```diff
@@ -0,0 +1,24 @@
+---
+import Layout from "../../../layouts/docs.astro";
+import IndexPage from "../../../components/docs/indexPage.astro";
+import { Frontmatter, SIDEBAR } from "../../../config";
+import { getLanguageFromURL } from "../../../languages";
+
+const frontmatter: Frontmatter = {
+  title: "Deployment",
+  layout: "docs",
+  description: "Aprenda como dar deploy na sua aplicação T3 para produção.",
```
Comment (Sn0wye) [inline www/src/pages/pt/deployment/index.astro:10]:
```suggestion
  description: "Aprenda como fazer deploy para produção na sua aplicação T3.",
```

File: www/src/pages/pt/usage/env-variables.md [hunk 7]
```diff
@@ -89,6 +89,8 @@ const dbUrl = env.DATABASE_URL;
 
 Como o arquivo `.env` padrão não está comprometido com o controle de versão, também incluímos um arquivo `.env.example`, no qual você pode, opcionalmente, manter uma cópia de seu arquivo `.env` com quaisquer segredos removidos. Isso não é necessário, mas recomendamos manter o exemplo atualizado para tornar mais fácil para os contribuidores começarem a usar seu ambiente.
 
+Alguns frameworks e ferramentas de construção, como o Next.js, sugerem que você armazene segredos em um arquivo `.env.local` e commita arquivos `.env` em seu projeto. Isso não é recomendado, pois poderia facilitar o acidente de commitar segredos em seu projeto. Em vez disso, recomendamos que você armazene segredos em `.env`, mantenha seu arquivo `.env` no seu `.gitignore` e commita somente arquivos `.env.example` em seu projeto.
```
Comment (Sn0wye) [inline www/src/pages/pt/usage/env-variables.md:92]:
```suggestion
Alguns frameworks e outras ferramentas, como o Next.js, sugerem que você armazene variáveis em um arquivo `.env.local` e faça commit de arquivos `.env` em seu projeto. Isso não é recomendado, pois poderia facilitar o acidente de  incluir variáveis ambiente secretas em seu histórico do git. Em vez disso, recomendamos que você armazene essas variáveis no arquivo `.env`, mantenha o arquivo `.env` em seu `.gitignore` e faça commit somente de arquivos `.env.example` em seu projeto.
```

File: www/src/pages/pt/usage/next-auth.md [hunk 8]
```diff
@@ -34,6 +34,29 @@ const User = () => {
 };
 ```
 
+## Recuperando a sessão do lado do servidor.
+
+Às vezes, você poderá querer solicitar a sessão no servidor. Para fazer isso, faça uma requisição usando o helper `getServerAuthSession` que `create-t3-app` fornece e passe-a para o cliente usando `getServerSideProps`:
```
Comment (Sn0wye) [inline www/src/pages/pt/usage/next-auth.md:39]:
```suggestion
Às vezes, você poderá querer solicitar a sessão no servidor. Para fazer isso, faça uma requisição usando o utilitário `getServerAuthSession` que `create-t3-app` fornece e passe-a para o cliente usando `getServerSideProps`:
```

File: www/src/pages/pt/usage/trpc.md [hunk 9]
```diff
@@ -127,6 +93,50 @@ const UserPage = () => {
 
 Você notará imediatamente como o preenchimento automático e a segurança de tipo são bons. Assim que você escrever `trpc.`, seus roteadores aparecerão no preenchimento automático, e quando você selecionar um roteador, seus procedimentos também aparecerão. Você também receberá um erro de TypeScript se sua entrada não corresponder ao validador definido no back-end.
 
+## Files
```
Comment (Sn0wye) [inline www/src/pages/pt/usage/trpc.md:96]:
```suggestion
## Arquivos
```

**Review by Sn0wye — CHANGES_REQUESTED** [review]
Just a few changes here and there, but looking great!

**Comment by gilhrpenner** [comment]
> Just a few changes here and there, but looking great!

Thanks 😄 
Been living outside of Brazil for more than 10 years, sometimes it's harder to translate a few tech terms I haven't used in the past lol

**Review by Sn0wye — APPROVED** [review]
All good! 🎉

**Review by juliusmarminge — APPROVED** [review]

**Comment by juliusmarminge** [comment]
Stale**

**Pull request merged**